.drone.yml

```diff
@@ -356,6 +347,6 @@ trigger:
 
 ---
 kind: signature
-hmac: 4af11df3fb7d9b47a79b7224b77be21e6c8e42118d6cab12f3522d9a76603a79
+hmac: 627b9b4069cb241c084037dfb0284fbe47d42dd52d2c0053d99a948fdd892abe
```

Comment:
What's this signature?

Reply:
It's this handy dandy feature: https://readme.drone.io/signature/

> You can optionally sign your configuration file to verify authenticity and prevent tampering. This is useful if your repository is public and you need to prevent unauthorized changes to your configuration.
\
If a user modifies the configuration and signature verification fails, the pipeline is blocked pending manual approval by an authorized user with write or administrative access to the repository.

This is very useful for us so that users who don't have write access can't modify the build scripts to dump secrets :lock:

Reply:
whoa sweet! Thanks for the link :slightly_smiling_face: